pkg/comp-functions/functions/vshnpostgres/postgresql_deploy.go

```diff
@@ -334,7 +334,7 @@ func createSgPostgresConfig(comp *vshnv1.VSHNPostgreSQL, svc *runtime.ServiceRun
 		},
 	}
 
-	err := svc.SetDesiredKubeObjectWithName(sgPostgresConfig, comp.GetName()+"-pgconf", "pg-conf")
+	err := svc.SetDesiredKubeObjectWithName(sgPostgresConfig, comp.GetName()+"-"+configResourceName, configResourceName)
```

Comment:
This will change the object name, which will probably result in the deletion of the old object.

I would keep the name as is.

Reply:
so after visiting this rabbit hole I think it's safe to proceed now